### PR TITLE
Fix issue #149 by removing from_string

### PR DESCRIPTION
**Pull request by boredom101**
This fixes issue #149 by removing from_string and adjusting the creation of Attributes. I am not sure about how I did nav_interface.
I still need to run the tests though, I will do it when I am in front of my computer, or someone else can do it.

### CODE REVIEW

**Comment by lapp0** [comment]
In `attribute.py` you should remove `def from_string`, and instead `def __init__(self, path)` should check whether the passed argument `isinstance(path, list)` or `isinstance(path, str)`. If its a list, `self.loc = path`, if its a string, use the logic from `def from_string`.

Then `grep -R "Attribute.from_string" nix-gui-path-here` and find all cases where it is used and replace it with `Attribute(args here)`

**Comment by boredom101** [comment]
I did it the right way this time, as well as running the tests. I had to use `__setattr__`, since it is a frozen set.

File: nixui/options/attribute.py [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-    @classmethod
-    def from_string(cls, attribute_string):
-        loc = next(csv.reader([attribute_string], delimiter='.', quotechar='"'))
-        return cls(loc)
+    def __init__(self, path):
+        if isinstance(path, list):
+            object.__setattr__(self, 'loc', path)
+        elif isinstance(path, str):
+            object.__setattr__(self, 'loc', next(csv.reader([path], delimiter='.', quotechar='"')))
```
Comment (lapp0) [inline nixui/options/attribute.py:14]: Recommend raising `TypeError` here. 

Looks good otherwise, nice work!

**Review by lapp0 — CHANGES_REQUESTED** [review]

File: nixui/options/attribute.py (hunk 1 above)
Comment (boredom101) [reply nixui/options/attribute.py:14]: Fixed this

**Review by lapp0 — APPROVED** [review]
`Attribute.from_string` removed per `grep -R `Attribute.from_string`

All tests pass per `nix flake check`.

If this is no longer a WIP, please remove the [WIP] label and I will merge.

Thanks for your great work!

**Comment by boredom101** [comment]
Sorry, forgot to do that.

**Pull request merged**